android/quest/src/main/java/org/smartregister/fhircore/quest/ui/login/LoginViewModel.kt

```diff
@@ -178,9 +181,25 @@ constructor(
     }
   }
 
-  fun forgotPassword() {
-    // TODO load supervisor contact e.g.
-    _launchDialPad.value = "tel:0123456789"
+  fun forgotPassword(context: Context) {
+    val contactNumber = applicationConfiguration.loginConfig.supervisorContactNumber
+    if (!contactNumber.isNullOrEmpty()) {
+      val formattedNumber = formatPhoneNumber(context, contactNumber)
+      _launchDialPad.value = formattedNumber
+    } else {
+      Toast.makeText(context, context.getString(R.string.call_supervisor), Toast.LENGTH_LONG).show()
+    }
+  }
+
+  fun formatPhoneNumber(context: Context, number: String): String {
+    return try {
+      val cleanedNumber = number.filter { it.isDigit() }
+      val countryCode = context.getString(R.string.country_code)
```

Comment:
The country code should not be hard coded.

Reply:
You can obtain the country code from `Locale.getDefault().country`

Then refactor the your code to use the country code 

```kotlin
  if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
    return PhoneNumberUtils.formatNumber(yourStringPhone,Locale.getDefault().getCountry());
  } else {
  //Deprecated method
    return PhoneNumberUtils.formatNumber(yourStringPhone); 
  }
```